docs/conf.py

```diff
@@ -170,6 +171,17 @@ def ext_candidates(fpath):
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = False
 
+# redoc config
+redoc = [
+    {
+        "name": "HELICS REST API",
+        "page": "references/api-reference/rest_queries_api",
```

Comment:
I think it would be better to generate this in the `swagger` folder.
```suggestion
        "page": "swagger",
```

And then also add a directive to RTD saying to include the contents of the `swagger` folder so that all the yaml/json specs and model files are copied to the website.

Reply:
@nightlark, If you think you know how to get this working, be my guest. I'm kind of guessing at how to get all the pathing right.

Reply:
I need to look into the redoc settings, but I think the original pathing and generated swagger files were fine. The two issues I think for why it didn't work are:

- RTD conf.py needs to specify that the `swagger` folder with model yaml files be included in the deployed site; embed is set to True for redoc, so it stores a copy of queries.yaml in the generated api page but the stored copy has paths referring to the other yaml files which aren't included on the RTD site
- embed should probably be left as the default value False, making so that the relative paths in queries.yaml point to the right place; when queries.yaml is embedded it looks like the generated in an arbitrary folder that breaks the relative paths

I'll see if I can try to get it working later today or tonight, if it isn't already working.

Reply:
I'll rollback to the an earlier version and try just turning `embed` off.

Reply:
I do think the `swagger` folder needs to end up in the `api-reference` folder but we can tweak that once we get things working.

Reply:
The swagger folder doesn't need to end up in `api-reference` (we'll do the same thing with it as we do the doxygen folder).

https://github.com/GMLC-TDC/HELICS/blob/swagger/docs/conf.py#L104
I think changing this line from:
```
html_extra_path = [os.path.abspath(os.path.join(doxygen_build_dir, "docs", "html"))]
```
to
```
html_extra_path = ["swagger", os.path.abspath(os.path.join(doxygen_build_dir, "docs", "html"))]
```

will include the swagger folder in the right place.

Reply:
From my understanding of how redoc works, it looks like reverting the changes back to commit [ded62ea](https://github.com/GMLC-TDC/HELICS/pull/2289/commits/ded62eae63cc76d02432ce27ba63226cc4ab0ebc), making the `html_extra_path` change, and turning `embed` off could work.

Reply:
Well, [this link](https://docs.helics.org/en/swagger/references/api-reference/rest_queries_api.html) still returns a 404. I'll see if I come up with any other great ideas.

Reply:
Yeah -- it looks like https://docs.helics.org/en/swagger/swagger/models/Filter.yaml doesn't exist, so I think it didn't copy the folder. Might be that the absolute path to the swagger folder needs to be given instead to make RTD include it... or relative paths are to something like the repository root instead rather than the docs subfolder.